### PR TITLE
feat(curriculum) create strings quiz

### PR DESCRIPTION
**Pull request by dineshkottaKota**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-javascript-strings/66edc31c44f1b9c1d5c5ebca.md [hunk 1]
```diff
@@ -17,243 +17,242 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+let str = Hello;
 
 ---
 
-Placeholder distractor 3
+let str = 'Hello';
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-javascript-strings/66edc31c44f1b9c1d5c5ebca.md:32]: this is not wrong, why it is here?

**Comment by ilenia-magoni** [comment]
You were not assigned to this quiz.

**Comment by dineshkottaKota** [comment]
@ilenia-magoni please don't label my PR as spam, this is my first hactoberfest and if you label it as spam i may disqualify from hactoberfest

**Comment by ilenia-magoni** [comment]
@dineshkottaKota you have opened 5 PRs so far, this one in which you totally ignored the instructions in the issue that specify that only who is assigned can work on the quizzes. And you opened #56650 after a moderator closed #56646. You will need to show you can contribute following guidelines, at that point we can think of removing the `spam` label.

**Comment by dineshkottaKota** [comment]
@ilenia-magoni I apologize for not following the guidelines correctly. I understand that only the person assigned to the issue should work on it, and I will make sure to follow this rule moving forward. I also acknowledge that opening PR #56650 after #56646 was closed was a mistake, and I'll be more careful in the future to respect decisions made by the moderators.

I appreciate your feedback and will take extra care to contribute within the guidelines. Please let me know if there's anything else I should address, and I'll work to improve.

Please don't label my PR as spam

**Comment by ilenia-magoni** [comment]
It's up to you if we will be able to remove the spam PR or not.

**Comment by dineshkottaKota** [comment]
@ilenia-magoni please consider my request and remove spam label and due to the two spam labels I was disqualified. 
Please accept my request

**Comment by ilenia-magoni** [comment]
@dineshkottaKota as I already said, your future behaviour will determine if we can remove the spam label or not

**Comment by dineshkottaKota** [comment]
@ilenia-magoni so what I have to do now?

**Comment by ilenia-magoni** [comment]
> You will need to show you can contribute following guidelines, at that point we can think of removing the `spam` label.

I already mentioned what it is expected from you @dineshkottaKota

**Pull request closed without merging**